core/src/main/java/feast/core/service/SpecService.java

```diff
@@ -277,6 +325,94 @@ public ListFeaturesResponse listFeatures(ListFeaturesRequest.Filter filter) {
     }
   }
 
+  /**
+   * Return a list of entities matching the entity name, project and labels provided in the filter.
+   * All fields are required. Use '*' in entity name and project, and empty map in labels in order
+   * to return all entities in all projects.
+   *
+   * <p>Project name can be explicitly provided, or an asterisk can be provided to match all
+   * projects. It is not possible to provide a combination of asterisks/wildcards and text. If the
+   * project name is omitted, the default project would be used.
+   *
+   * <p>The entity name in the filter accepts an asterisk as a wildcard. All matching entities will
+   * be returned. Regex is not supported. Explicitly defining an entity name is not possible if a
+   * project name is not set explicitly.
+   *
+   * <p>The labels in the filter accepts a map. All entities which contain every provided label will
+   * be returned.
+   *
+   * @param filter Filter containing the desired entity name, project and labels
+   * @return ListEntitiesResponse with list of entities found matching the filter
+   */
+  public ListEntitiesResponse listEntities(ListEntitiesRequest.Filter filter) {
```

Comment:
Do we have a strong use for this wildcard filters or can we remove it?

Reply:
I guess it'll be easier to locate a set of entities that are registered with a certain pattern within a project.

Reply:
Yes, but is there a need for it?

Reply:
Not a strong need, have removed it.

Reply:
Hmmm, what have you removed? I still see `name.replace('*', '%'), project.replace('*', '%'))`

Reply:
If its easier to keep in then we can keep it in, but this seems like a lot of complexity to keep if we dont actually need it.

Reply:
Oh, I disallowed searching for wildcard names in a specific project. So if we're removing wildcard filters, should we update the proto to only consider project and labels in the Filter then?

Reply:
Doesn't seem very useful to have entity name in the labels since it'll just return that entity in a list. The user might be better off using getEntity in this case.